index.js

```diff
@@ -159,22 +159,24 @@ var vueTouchEvents = {
 
             } else if (!$this.swipeOutBounded) {
                 var swipeOutBounded = $this.options.swipeTolerance,
-                    direction;
+                    direction,
+                    distanceY = Math.abs($this.startY - $this.currentY),
+                    distanceX = Math.abs($this.startX - $this.currentX);
 
-                if (Math.abs($this.startX - $this.currentX) < swipeOutBounded) {
-                    direction = $this.startY > $this.currentY ? 'top' : 'bottom';
-
-                } else {
-                    direction = $this.startX > $this.currentX ? 'left' : 'right';
-                }
-
-                // Only emit the specified event when it has modifiers
-                if ($this.callbacks['swipe.' + direction]) {
-                    triggerEvent(event, this, 'swipe.' + direction, direction);
+                if (distanceY > swipeOutBounded || distanceX > swipeOutBounded) {
```

Comment:
When the code runs into the code block of L160,  it means a `swipe` event is detected. This event only has two possible directions, top-bottom and left-right. So this judgement has no differece to the old version.

Reply:
I should have originally supplied an example, sorry:

`<template>
    <div class="swipe" v-touch:swipe.bottom="swipeHandler" v-touch-options="{swipeTolerance: 9999}">
      <br>
      <br>
      <br>
      <br>
      <br>
      <br>
      <br>
      <br>
    </div>
</template>

<script>
export default {
  name: 'VueSwipeTest',
  methods: {
    swipeHandler(e) {
      alert('swipe');
    }
  }
}
</script>

<style scoped>
  .swipe {
    background: red;
  }
</style>
`

When swiping down swipeHandler() will fire disregarding the swipeTolerance